api/helpers/common_helper.py

```diff
@@ -9,6 +9,29 @@ def sanitize_phone_string(phoneString):
 
 
 def fetch_by_key(key, data):
+    """Checks whether a key is present in a json array
+
+    Args:
+        key (string): The key which we will be searching for
+        data (json): The json array in which we are going to make the search
+
+    Returns:
+        string: It will return either string or null if not available
+    """
     if key in data:
         return data[key]
     return None
+
+
+def string_contains_list_item(input_string, list_data):
```

Comment:
Can you rename the function to something like `list_having_string`

Reply:
Renamed.